### ISSUE TITLE
The 5.14.0 release contains a fatal issue in combination with Typescript

### ISSUE THREAD

**Issue by rickprent** [Adyen/adyen-web#1592]
**Describe the bug**
Our CI is failing on the latest 5.14.0 release with the following error:
```
Error: node_modules/@adyen/adyen-web/dist/types/components/internal/Address/Specifications.d.ts:2:30 - error TS2307: Cannot find module 'src/types' or its corresponding type declarations.
2 import { AddressField } from 'src/types';
                               ~~~~~~~~~~~
```
My guess is that the added import path in https://github.com/Adyen/adyen-web/commit/f263280f96350dfa3776aaf9313b027de57542f7#diff-5070a5d327dd38b28708885484f1d987343e9f2495c56c0fc6eb5277762f0dbdR3 should be relative instead of absolute.
The absolute path somehow conflicts with Typescript.

I've made a public test GitHub repository to reproduce the bug: https://github.com/rickprent/adyen-web-5.14.0-test
Yarn version: 1.22.18

**To Reproduce**
1. Checkout https://github.com/rickprent/adyen-web-5.14.0-test
2. Run `yarn install`
3. Run `yarn tsc index.ts`

**Expected behavior**
No CLI error to show up.

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04.4 LTS

**Comment by ribeiroguilherme** [comment]
Hi @rickprent ,

Thanks for reporting it to us. This issue was fixed on 5.15.0 and it is already available 👌 Cheers!

**Comment by rickprent** [comment]
Nice! Thanks for the fix @ribeiroguilherme.
